### PR TITLE
WebDriver conformance updates for Firefox 96 release notes

### PR DESCRIPTION
**Pull request by whimboo**
Related changes for Marionette in Firefox 96 can be found [in this query](https://bugzilla.mozilla.org/buglist.cgi?j_top=OR&o1=equals&resolution=FIXED&o2=equals&query_format=advanced&v1=fixed&component=Marionette&v2=verified&product=Testing&f1=cf_status_firefox96&list_id=15958615).

@juliandescottes could you please pre-review? Thanks.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/96/index.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ No notable changes.
 - A number of deprecated non-standard statistics fields have been removed from the [WebRTC Statistics API](/en-US/docs/Web/API/WebRTC_Statistics_API), including: `bitrateMean`, `bitrateStdDev`,`framerateMean`, `framerateStdDev`, and `droppedFrames`.
   ({{bug(1367562)}}).
 
+### WebDriver conformance (Marionette)
+
+- Added the command `WebDriver:GetElementShadowRoot` to retrieve the root node of a DOM subtree for a given element ({{bug(1700073)}}).
```
Comment (juliandescottes) [inline files/en-us/mozilla/firefox/releases/96/index.md:67]: I imagine "root node of a DOM subtree" is from https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot, but without the last bit "that is rendered separately from a document's main DOM tree", it is misleading.

Also the command only returns something if the element hosts a shadow root. Meaning it won't return anything if you call it on an element inside the shadow DOM subtree. So I would rephrase this closer to https://developer.mozilla.org/en-US/docs/Web/API/Element/shadowRoot
eg
```
Added the command `WebDriver:GetElementShadowRoot` to retrieve the shadow root (open or closed) hosted by a given element ({{bug(1700073)}}).
```

**Review by juliandescottes — COMMENTED** [review]
One suggestion for the shadow root feature.

File: files/en-us/mozilla/firefox/releases/96/index.md (hunk 1 above)
Comment (whimboo) [reply files/en-us/mozilla/firefox/releases/96/index.md:67]: Yes, and that sounds fair. Will update my PR with your suggestion. Thanks for having a look!

**Comment by whimboo** [comment]
@Rumyra or @sideshowbarker could one of you please do the final review and get it merged? Thanks.

**Review by sideshowbarker — APPROVED** [review]
